CHANGELOG.md

```diff
@@ -2,83 +2,88 @@
 
 ## Unreleased
 
+## 1.0.0
```

Comment:
I am not really comfortable with immediately going v1 here - although I recognize that this introduces a breaking change so it might be prudent to do so.

Reply:
Sure, I'd take your direction on this, would you make this version 0.6.0 instead? I guess there's implicit instability in the version being less than 1, but I did recently have an experience with a library introducing a breaking change in a minor version, and it made what should have been a quick/simple update of our dependencies, turn into a protracted debugging task.

Reply:
I mean, technically I already crossed the rubicon when I rewrote the lib between 0.1 and 0.2 - I think 0.6.0 would be fine actually, since I'd want more of the Delivery APIs covered for a v1.0.0.